src/Entity/User.php

```diff
@@ -35,6 +42,58 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
      */
     private string $password;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     * @Assert\NotBlank(
+     *     message = "Le prénom est obligatoire")
+     */
+    private string $firstname;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     * @Assert\NotBlank(
+     *     message = "Le nom de famille est obligatoire")
+     */
+    private string $lastname;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     * @Assert\Choice({"female", "male", "non binary", null})
+     */
+    private string $gender;
+
+    /**
+     * @ORM\Column(type="integer")
+     * @Assert\NotBlank(
+     *     message = "L'âge est obligatoire")
+     * @Assert\GreaterThan(
+     *     value = 17, message = "Vous devez être majeur.e pour bénéficier du service")
+     * @Assert\LessThan(
+     *      value = 25,
+     *      message = "Vous ne pouvez pas être âgé.e de plus de {{ compared_value }} pour bénéficier du service")
+     */
+    private int $age;
+
+    /**
+     * @ORM\Column(type="string")
+     * @Assert\NotBlank(
+     *     message = "Le numéro de téléphone est obligatoire")
+     * @Assert\Regex(
+     *     pattern="/[0-9]{9}/",
+     *     message = "Veuillez entrer un numéro de téléphone valide")
```

Comment:
You will have valid numbers considered invalid using this regex.
For example +33600000000 is valid
And some people are used to have separators in phone number like . spaces or -
You may use a less strict constraints here like... between 10 to 20 characters ?